notebooks/equivalence_equiangular_sampling.ipynb

```diff
@@ -0,0 +1,375 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "76430a19-e222-4c2c-9e4e-743641a5e12e",
+   "metadata": {},
+   "source": [
+    "# Equivalence of various equiangular samplings\n",
+    "\n",
+    "In this demo we show the equivalence between Fejer/Clenshaw-Curtis sampling and other equiangular sampling schemes, including MW, MWSS and DH."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "be9189dc-7827-416a-9340-fb66fbc390e1",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "JAX is not using 64-bit precision. This will dramatically affect numerical precision at even moderate L.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import s2fft \n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81b92ac3-36e8-40fc-93f6-45394eb673dd",
+   "metadata": {},
+   "source": [
+    "Set band-limit for tests."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "463a7044-99c3-45e6-bce2-3b3c7dd0e259",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "L = 4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f59ae250-b286-4a9a-8a45-32098fa134a1",
+   "metadata": {},
+   "source": [
+    "## Equivalence of MW and Fejer type 1 sampling"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "8559d609-67d5-4561-bb43-2de124f49e25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "thetas_mw = s2fft.sampling.s2_samples.thetas(L, sampling=\"mw\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "bed1f19f-d015-4284-ade6-eeb8954d34b9",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([0.44879895, 1.34639685, 2.24399475, 3.14159265])"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "thetas_mw"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "ad22a641-3940-489e-98f5-315f0ca4c5e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n = L - 1/2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "de96933f-6b89-4796-bdcd-1eb4959a1a55",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "thetas_mw_fejer1 = (np.arange(0, n) + 0.5) * np.pi / n"
```

Comment:
It would be useful to have a reference for this formula or at least some explanation of how it arises.